openshift/keycloak.app.yaml

```diff
@@ -103,15 +109,25 @@ objects:
               secretKeyRef:
                 name: keycloak-config
                 key: db.port
+          - name: OPENSHIFT_KUBE_PING_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: OPENSHIFT_KUBE_PING_LABELS
+            value: 'name=keycloak-server'
+          - name: OPENSHIFT_KUBE_PING_SERVER_PORT
+            value: '47600'
           - name: JAVA_OPTS
             value: >-
-              -server -Xms256m -Xmx4096m -XX:MetaspaceSize=96M
+              -server -Xms256m -Xmx3584m -XX:MetaspaceSize=96M
```

Comment:
What about dumping heaps on error? We dont need that anymore? Is that issue solved?